R/mod_review_forms.R

```diff
@@ -110,14 +110,10 @@ mod_review_forms_server <- function(
     ns <- session$ns
     
     review_data_active <- reactive({
-      df <- r$review_data |>
+      r$review_data |>
         dplyr::filter(subject_id == r$subject_id, 
                       item_group == active_form()) |> 
-        dplyr::distinct(subject_id, item_group, edit_date_time, reviewed, comment, status)
-      #!! below selects the latest edit_date_time; usually only one row will remain by then since there are no items displayed here.
-      if(nrow(df)== 0) return(df)
-      df |> 
-        dplyr::filter(edit_date_time == max(as.POSIXct(edit_date_time)))
+        dplyr::select(id, dplyr::all_of(idx_cols), edit_date_time, reviewed, comment, status)
```

Comment:
Is this select statement still needed? Or can we just leave it out?

Reply:
I will remove it.